src/bin/typescript-coverage-report.ts

```diff
@@ -94,6 +94,10 @@ program
     "allow writes to variables with implicit any types",
     ignoreUnread
   )
+  .option(
+    "-- [string[]]",
```

Comment:
Why `--` and not `--files`?

Reply:
--files wont let you pass multiple paths like "typescript-coverage-report -- src/lib/getCoverage.ts src/lib/index.ts", it will only take src/lib/getCoverage.ts and give it as string, not string[], lint staged uses -- to pass files

Reply:
I see! Thanks @StaRenn